TransformRecorder/qSlicerTrackedSequenceBrowserWriter.cxx

```diff
@@ -315,14 +314,30 @@ ::writeSQBR( vtkMRMLSequenceBrowserNode* trackedSequenceBrowserNode, std::string
     {
       continue;
     }
-    vtkSmartPointer< vtkMRMLNode > currTempProxyNode = currProxyNode->CreateNodeInstance();
+    vtkSmartPointer<vtkMRMLNode> currTempProxyNode = vtkSmartPointer<vtkMRMLNode>::Take(currProxyNode->CreateNodeInstance());
     currTempProxyNode->Copy( currProxyNode );
     currTempProxyNode->SetScene( tempScene.GetPointer() );
     tempScene->AddNode( currTempProxyNode );
     tempTrackedSequenceBrowserNode->AddProxyNode( currTempProxyNode, currTempSequenceNode, false );
 
     this->copyNodeAttributes( currProxyNode, currTempProxyNode );
     currTempProxyNode->UpdateReferences();
+
+    if (currSequenceNode->GetStorageNode())
+    {
+      // Use the same class of sequence storage node that is used in the scene
+
+      vtkSmartPointer<vtkMRMLStorageNode> currSequenceStorageNode = currSequenceNode->GetStorageNode();
+      vtkSmartPointer<vtkMRMLStorageNode> currTempSequenceStorageNode = vtkSmartPointer<vtkMRMLStorageNode>::Take(vtkMRMLStorageNode::SafeDownCast(currSequenceStorageNode->CreateNodeInstance()));
+      currTempSequenceStorageNode->Copy(currSequenceStorageNode);
+      currTempSequenceStorageNode->SetScene(tempScene.GetPointer());
+      tempScene->AddNode(currTempSequenceStorageNode);
+
+      this->copyNodeAttributes(currSequenceStorageNode, currTempSequenceStorageNode);
```

Comment:
I think Copy copies node attributes as well. Is this needed?

Reply:
I don't know if it's needed, but we already do this for the sequence nodes.

Is there a reason why we shouldn't copy attributes?

Reply:
Yes, I just meant that this is probably not needed because Copy or CopyContent copies the attributes already.